serverless/src/getContactInfo/handler.js

```diff
@@ -31,20 +33,46 @@ const getContactInfo = async (event, context) => {
   try {
     const userRecord = await dbConnection('users')
       .first(
-        'cmr_preferences',
+        'urs_id',
         'urs_profile'
       )
       .where({
         id
       })
 
+    const {
+      access_token: authToken
+    } = await getAccessTokenFromJwtToken(jwtToken, earthdataEnvironment)
+
+    // Make a request to CMR-ordering instead of the EDSC database
+    const cmrPreferencesData = await getCmrPreferencesData(
+      userRecord.urs_id,
+      authToken,
+      earthdataEnvironment
+    )
+    const { status, data, errors } = cmrPreferencesData
```

Comment:
This is the wrong place to pull `errors` out. `status` and `data` here are returned by axios. Within the `data` object is the response from cmr-ordering, which will have `data` and/or `errors`. 

To see what the response with errors will look like, log out `cmrPreferencesData` here, and within `getCmrPreferencesData`, overwrite the `ursId` being sent as a variable in the graphql call. 
```
    data: {
      variables: {
        ursId: `${username}asdf`
      },
      query: cmrQuery
    },
```

Your logged out cmrPreferencesData will have this 
```
  data: { errors: [ [Object] ], data: { user: null } }
```